src/styles/_catalog.scss

```diff
@@ -1,22 +1,37 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  &__content {
+    display: grid;
+    gap: 48px;
+    justify-items: center;
+    max-width: 944px;
+    margin: 0 auto;
 
-  max-width: 944px;
-  margin: 0 auto;
+    @media (min-width: 488px) {
+      grid-template-columns: repeat(2, 1fr);
+    }
```

Comment:
You can set grid-template-columns to the general class instead of one additional media query